### ISSUE TITLE
accuracy lower than that in the paper

### ISSUE THREAD

**Issue by YamengGu** [eeyhsong/EEG-Conformer#30]
Hi,

I edited the input data from dataset 2a which included both data and label. I run the code and found that each subject's accuracy is lower than that you described in the paper. What should I do? for example, my subject 1 best accuracy is 77% while yours is 88. my second subject is 47% while yours is 61%. other subjects keep the same trend.

Here is the link for my input data, just in case my input data is wrong: https://drive.google.com/file/d/1WQaB4G2i6z8r6drgbJ9gF5aZCrA_Ydek/view?usp=sharing

Thanks,

Yameng

**Comment by eeyhsong** [comment]
Do you use the example data I supported in the google driver?

**Comment by YamengGu** [comment]
No. I used the data i created in the shared Google drive. It is very similar to the 01E data you shared.Best,YamengSent from my iPhoneOn Apr 14, 2024, at 10:34 AM, eeyhsong ***@***.***> wrote:﻿
Do you use the example data I supported in the google driver?

**Comment by eeyhsong** [comment]
Where did you get the data? Did you perform some preprocessing such as band-pass filter and z-score?
refer to [code](https://github.com/eeyhsong/EEG-Transformer/blob/main/getData.m)

Best,
Song
